### PR TITLE
parse inline string values

### PR DESCRIPTION
**Pull request by kenichi**
* set value_type to :inline_string on <is> tag start
* capture cdata value when :inline_string
* test for values in existing test file

### CODE REVIEW

**Comment by kenichi** [comment]
slightly different than #108 which I probably would've just used if i'd looked first 😂

**Comment by RodolfoSilva** [comment]
@jsonkenl could you merge this fix?

**Review by RodolfoSilva — APPROVED** [review]
LGTM!

File: lib/xlsxir/parse_worksheet.ex [hunk 1]
```diff
@@ -87,10 +87,14 @@ defmodule Xlsxir.ParseWorksheet do
...
   end
 
+  def sax_event_handler({:startElement, _, 'is', _, _}, state, _, _),
+    do: %{state | value_type: :inline_string}
+
   def sax_event_handler({:characters, value}, state, _, _) do
     case state do
       nil -> nil
       %{value_type: :value} -> %{state | value: value}
+      %{value_type: :inline_string} -> %{state | value: value}
```
Comment (RodolfoSilva) [inline lib/xlsxir/parse_worksheet.ex:97]: I think you don't need this, just use the value_type: :value.
```suggestion
```

File: lib/xlsxir/parse_worksheet.ex [hunk 2]
```diff
@@ -87,10 +87,14 @@ defmodule Xlsxir.ParseWorksheet do
     %{state | value_type: :value}
   end
 
+  def sax_event_handler({:startElement, _, 'is', _, _}, state, _, _),
+    do: %{state | value_type: :inline_string}
```
Comment (RodolfoSilva) [inline lib/xlsxir/parse_worksheet.ex:91]: I believe we can use the type value here, or change the previous method implementation to use guards instead a constant.

```suggestion
    do: %{state | value_type: :value}
```

**Comment by RodolfoSilva** [comment]
Here the spec:

https://learn.microsoft.com/en-us/dotnet/api/documentformat.openxml.spreadsheet.inlinestring?view=openxml-2.8.1#remarks

**Comment by jsonkenl** [comment]
@kenichi Please resolve conflicts and I'll get this merged.

**Comment by kenichi** [comment]
@RodolfoSilva thanks for the suggestions, @jsonkenl resolution incoming...

**Pull request merged**